pillar/registry.sls

```diff
@@ -1,3 +1,29 @@
+network:
+  host_id: ocp13
```

Comment:
Right now mailname is amy01.open-contracting.org and host files have both ocp13 and amy01. Do we want to change it with this commit?

Reply:
Agreed, let's make this change on it's own PR.
This PR sets the groundwork for #278 but doesn't have to implement it.